sanic/response.py

```diff
@@ -218,7 +222,7 @@ def output(
                    b'keep-alive' if keep_alive else b'close',
                    timeout_header,
                    headers,
-                   self.body
+                   self.body if self.status is not 204 else b''
```

Comment:
Instead of this can you do the same thing you did with content-length? So, move a variable `body = ''` up there to line ~200 and in `if self.status is not 204:` set `body=self.body`. Finally use `body` here instead of `self.body`

Reply:
That way we do only one check of the status code instead of 2

Reply:
@r0fls It's not bug, just fix content-length = 0 for status code is 204

Reply:
@r0fls sure, will do! thanks for the feedback :)